cmd/cloud/cluster_flag.go

```diff
@@ -6,16 +6,19 @@ import (
 
 func setClusterFlags(command *cobra.Command) {
 	command.PersistentFlags().String("server", defaultLocalServerAPI, "The provisioning server whose API will be queried.")
+	command.PersistentFlags().String("token", "", "The auth token to access provisioning server.")
```

Comment:
Let's change this to accept any number of header keys+values. I believe `Flags().StringToStringVar` may be what we want here.

This way we can specify new headers or change the token key if we want.

Reply:
Yep seems more valid since we provide the option . 
I will rename the flag to `--extra-headers` WDYT ?

Reply:
The name you selected (`headers`) LGTM